testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -755,8 +781,8 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
     createRawCoinbaseTransaction(sender, receiver, amount)
       .flatMap(signRawTransaction(sender, _))
       .flatMap { signedTransaction =>
-        sender
-          .generate(100)
+        sender.getNewAddress.flatMap(sender
+          .generateToAddress(100, _))
           .flatMap { _ => // Can't spend coinbase until depth 100
```

Comment:
That is a lot of `flatMap`s, totally my fault but would you mind changing this to be a `for` without any `flatMap`s?